components/reference-data/src/main/java/org/datacleaner/beans/transform/RemoveDictionaryMatchesTransformer.java

```diff
@@ -135,6 +135,12 @@ public OutputColumns getOutputColumns() {
     @Initialize
     public void init() {
         dictionaryConnection = _dictionary.openConnection(_configuration);
+        dictionary = new ArrayList<>();
```

Comment:
Naming wise I think it would make sense to call this "dictionaryValues" or so.
And shouldn't it be sorted so that the longest one is the first and so on?